aspnetcore/security/authentication/individual.md

```diff
@@ -32,6 +32,17 @@ dotnet new razor -au Individual
 
 ::: moniker-end
 
+## No Authentication
+
+Authentication is specified in the .NET Core CLI with `-au` option. In Visual Studio, the **Change Authentication** dialog is available for new web applications. The default for new web apps in Visual Studio is **No Authentication**.
+
+Projects created with no authentication:
```

Comment:
Suggestion: "with no" :point_right: without

Reply:
It's bold **No Authentication** because that's the text the VS UI uses. This section is for a fwlink to **No Authentication**.

Reply:
I mean the "Projects created with no authentication:" part.

It could go "Projects created without authentication:" ... but as you say, it won't match.